generate/Cargo.toml

```diff
@@ -9,9 +9,9 @@ license-file = "../../LICENSE"
 typing = []
 
 [dependencies]
-iroha_schema = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-dev" }
-iroha_schema_gen = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-dev" }
-iroha_data_model  = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-dev" }
+iroha_schema = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-stable" }
+iroha_schema_gen = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-stable" }
+iroha_data_model  = { git = "https://github.com/hyperledger/iroha", branch = "iroha2-stable" }
```

Comment:
I'd recommend using `dev` for now, given that we have a release coming up soon.

Reply:
Will add in separate PR, since it requires changes to rust lib.